modules/mri_violations/php/mri_violations.class.inc

```diff
@@ -294,18 +296,12 @@ class Mri_Violations extends \NDB_Menu_Filter_Form
                    )
                 ) as hash,
                 MRICandidateErrors.ID as join_id,
-                p.CenterID as Site,
+                null,
```

Comment:
@cmadjar Could you explain why lines 285-286 and 299 were changed to null?

Reply:
Shouldn't a join return a `NULL` value anyways if the value doesn't exist? But that does make sense. Thanks for the clarification!

Reply:
What @um4r12 said.. wouldn't a join return a null value anyways?

Reply:
I don't see the point of having a JOIN here even if it returns a NULL value. In this case, we don't know the PSCID or CandID or Visit label so why trying to join to the session or candidate table when you know you'll get a NULL value?

Reply:
Well, for some errors you get a subprojectID and ProjectID with the assumption the PSCID is correct (but you have PSCID and CandID not matching so it might not be information for the correct candidate). 
Leigh agreed with me that it is better to have no information than information that could be wrong when a scan has not been properly associated with a session and candidate, hence the hardcoded NULL.